ux_research.md

```diff
@@ -0,0 +1,89 @@
+
+There are many methods for studying usability, but the most basic and useful is Usability Testing, which has 3 components:
+
+- Recruit representative users.
+- Ask users to perform representative tasks with the design.
+- Observe (and sometimes record) what the users do, where they succeed, and where they have difficulties with the user interface. The only interaction we should have with the user is to ask them to complete a task, or ask them to describe their thinking and reasoning out loud as they interact with the UI to work through the task.
+
+It's important to test users individually and let them solve any problems on their own. If we help them or direct their attention to any particular part of the screen, we have contaminated the test results.
+
+To identify a design's most important usability problems, [testing 5 unique users](https://www.nngroup.com/articles/why-you-only-need-to-test-with-5-users/) is typically enough. It's a better use of resources to run many small tests and revise the design between each one so we can fix the usability flaws as we identify them, rather than scheduling one or two big expensive studies each year. Iterative design is the best way to increase the quality of user experience. The more versions and interface ideas you test with users, the better. Finally, listening to what people say is often misleading: you have to watch what they actually do.
+
+### Recruiting and Screening Research Candidates
+
+Although there are many ways to recruit test candidates, at DLS we have two available to us, maintaining a pool of volunteers and "hallway recruiting". No matter which method is selected for a project, make sure to disclose that we intend to record the test session before scheduling. If a prospective participant is not comfortable with being recorded, it would be best to find another participant.
+
+#### Pool of existing participants
+
+It makes sense for us to recruit from our existing users and staff to establish a pool of volunteers who are willing to either try out new features or participate in research opportunities in general. One challenge our team faces is that we are not able to compensate these testers for their labor. To remedy this problem, we can look to the large pool of staff and student workers who are presently compensated for the tasks they are asked to perform at the Library, and perhaps even collaborate to establish a network of potential test candidates at the University. We will need the permission from PUL leadership to pursue this method of recruitment. Using a pool of existing participants is best for more formal testing of a larger set of tasks.
```

Comment:
Do we still need to consult PUL leadership? If so this seems like a todo item rather than something to record in the handbook.

Reply:
Yes, I will remove it.